dotnet-desktop-guide/net/winforms/printing/overview.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "PrintDialog Component Overview"
+ms.description: "Learn about print related settings in Windows-based applications."
+ms.date: "04/13/2022"
+f1_keywords:
+  - "PrintDialog"
+helpviewer_keywords:
+  - "Print dialog box [Windows Forms], displaying"
+  - "PrintDialog component [Windows Forms], about PrintDialog component"
+ms.custom: devdivchpfy22
+---
+
+# PrintDialog Component Overview (Windows Forms)
+
+The Windows Forms [PrintDialog](../../../framework/winforms/controls/printdialog-component-windows-forms.md) component is a pre-configured dialog box used to select a printer, choose the pages to print, and determine other print-related settings in Windows-based applications. Use it as a simple solution for printer and print-related settings selection in lieu of configuring your own dialog box. You can enable users to print many parts of their documents: print all, print a selected page range, or print a selection. By relying on standard Windows dialog boxes, you create applications whose basic functionality is immediately familiar to users. The <xref:System.Windows.Forms.PrintDialog> component inherits from the <xref:System.Windows.Forms.CommonDialog> class.
+
+## Working with the Component
+
+Use the <xref:System.Windows.Forms.CommonDialog.ShowDialog%2A> method to display the dialog box at run time. This component has properties that relate to either a single print job (<xref:System.Drawing.Printing.PrintDocument> class) or the settings of an individual printer (<xref:System.Drawing.Printing.PrinterSettings> class). Either of these, in turn, may be shared by multiple printers.
+
+When it is added to a form, the <xref:System.Windows.Forms.PrintDialog> component appears in the tray at the bottom of the Windows Forms Designer in Visual Studio.
+
+## How to: Capture User Input from a PrintDialog at Run Time
+
+While you can set options related to printing at design time, you will sometimes want to change these options at run time, most likely because of choices made by the user. You can capture user input for printing a document using the <xref:System.Windows.Forms.PrintDialog> and the <xref:System.Drawing.Printing.PrintDocument> components.  
+  
+### To change print options programmatically  
+  
+1. Add a <xref:System.Windows.Forms.PrintDialog> and a <xref:System.Drawing.Printing.PrintDocument> component to your form.  
```

Comment:
It's easier to make sure that child content aligns correctly in markdown if the items start with `01. ` I would convert them all to that format.
```suggestion
01. Add a <xref:System.Windows.Forms.PrintDialog> and a <xref:System.Drawing.Printing.PrintDocument> component to your form.  
```

Reply:
ok. done.